templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 5.0
 
+**Feature level 112**
+
+* [`GET /messages/ids`](/messages/ids): Added an api endpoint to
```

Comment:
This CI failure is here: Your link should be to `/api/get-messages-from-message-ids`, since that's the `operationId` for the endpoint.

Though we probably want a different `operationId` anyway.